### ISSUE TITLE
DEBUG_SOCKETS seems to always be on

### ISSUE THREAD

**Issue by wcarthur1** [01org/TPM2.0-TSS#284]
I edited the default_config.site file to remove the -DDEBUG_SOCKETS flag from it.  And then rebuilt everything.  Still seeing all the socket debug messages.  
This only happens on Linux builds.  Windows works as intended.

**Comment by wcarthur1** [comment]
Some more data on this:  I pulled the latest commit, f9d80f7, and rebuilt the debug version (which had -DDEBUG_SOCKETS set).

This gave me NO debug messages at all, regardless of whether I set the debug level or not.

Then, because I was perplexed as to why I was suddenly getting no debug messages at when before I was getting all of them, I built with -DDEBUG_SOCKETS removed from the debug_config.site file.  Now I get ALL the debug messages, both the normal ones and the DEBUG_SOCKETS ones.  And my command line debug setting makes no difference in the amount of debug messages.
